pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -546,106 +554,141 @@ func TestSimpleDaemonSetScheduleDaemonSetPodsLaunchesPods(t *testing.T) {
 // Simulate a cluster with 100 nodes, but simulate a limit (like a quota limit)
 // of 10 pods, and verify that the ds doesn't make 100 create calls per sync pass
 func TestSimpleDaemonSetPodCreateErrors(t *testing.T) {
-	for _, strategy := range updateStrategies() {
-		ds := newDaemonSet("foo")
-		ds.Spec.UpdateStrategy = *strategy
-		manager, podControl, _, err := newTestController(ds)
+	var ScheduleDSPods = "ScheduleDaemonSetPods"
+	for _, f := range []bool{true, false} {
```

Comment:
@janetkuo - Please let me know, if this is what you wanted. Basically testing when testing is the featuregate is enabled and when it is disabled.

Reply:
Yes, thanks